solutions/functions/swap-function.js

```diff
@@ -0,0 +1,27 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+  idx++;
+  return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+function swapFunction(first, second){
+    let temp = first;
+    first = second;
+    second = temp;
+
+    return [first, second];
+}
+
+let firstNumber = parseInt(readLine());
+let secondNumber = parseInt(readLine());
+
+[firstNumber, secondNumber] = swapFunction(firstNumber, secondNumber);
```

Comment:
I don't think `swap` is the right problem for functions. It would have been right in `C` or `C++`. In `JS`, it seems very artificial. Various reasons.

1. Function could very well have been `return [second, first]`. No need of temp and all.
2. Even better, the RHS could be `[second, first]`.

I recommend not giving this as a practice problem at all.